packages/website/styles/ui-components/Chakra-Button.tsx

```diff
@@ -0,0 +1,156 @@
+import { Button, ButtonGroup } from '@chakra-ui/react';
+
+type ChakraButtonProps = {
+  children?: React.ReactNode;
+  onClick?: (event: React.MouseEvent) => void;
+  as?: React.ElementType;
+};
+
+export const ButtonBlack = ({ children, as, onClick }: ChakraButtonProps) => {
+  return (
+    <Button
+      size="sm"
+      bgColor="#000000"
+      _hover={{ cursor: 'pointer', bg: '#30363D' }}
+      textColor="white"
+      m="5px"
+      fontSize="14px"
+      onClick={onClick}
+      as={as}
+    >
+      {children}
+    </Button>
+  );
+};
+
+export const ButtonGray = ({ children, as, onClick }: ChakraButtonProps) => {
+  return (
+    <Button
+      bgColor="#EDF2F7"
+      textColor="black"
+      _hover={{ cursor: 'pointer', bg: '#E2E9F0' }}
+      m="5px"
+      ms="0px"
+      w="auto"
+      fontSize="14px"
+      onClick={onClick}
+      as={as}
+    >
+      {children}
+    </Button>
+  );
+};
+
+export const ButtonGray2 = ({ children, as, onClick }: ChakraButtonProps) => {
```

Comment:
idk if this is new or moved, but `ButtonGray2` should be renamed to what's different about it vs `ButtonGray`